src/apis/AppState/index.js

```diff
@@ -2,7 +2,7 @@ import ExecutionEnvironment from 'fbjs/lib/ExecutionEnvironment';
 import findIndex from 'array-find-index';
 import invariant from 'fbjs/lib/invariant';
 
-const EVENT_TYPES = [ 'change' ];
+const EVENT_TYPES = ['change'];
```

Comment:
Same, I find it harder to read without spaces. I guess people will complain either way but I was expecting spacing to be added for both `{` and `[` for consistency.